runFlask.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+VAG_COMMON_DIR="/vagrant"
+HOME="/home/vagrant"
+LOG_DIR="$VAG_COMMON_DIR/log"
+FLASK_APP_PY="$VAG_COMMON_DIR/flask_init.py"
+FLASK_DIR=$HOME/FlaskApp
+
+cd $FLASK_DIR
+if [ -e venv/bin/activate ]; then
```

Comment:
Why a virtualenv was required?

Reply:
In short, a best practice to isolate each Flask project with its own Python libraries to ensure all dependencies.

Taken from Flask Documentation
https://flask.palletsprojects.com/en/1.1.x/installation/

"The more Python projects you have, the more likely it is that you need to work with different versions of Python libraries, or even Python itself. Newer versions of libraries for one project can break compatibility in another project.

Virtual environments are independent groups of Python libraries, one for each project. Packages installed for one project will not affect other projects or the operating system’s packages."

Reply:
@DudiChen The documentation is correct, but, not relevant to case of spining a DEV environment with vagrant. It is by itself an isolated environment.